src/chttpd/src/chttpd_stats.erl

```diff
@@ -24,29 +24,38 @@
     incr_writes/1,
 
     incr_rows/0,
-    incr_rows/1
+    incr_rows/1,
+
+    update_interval/1
 ]).
 
 
 -record(st, {
     reads = 0,
     writes = 0,
-    rows = 0
+    rows = 0,
+    last_report_ts = 0,
+    interval,
+    request
 }).
 
 
 -define(KEY, chttpd_stats).
+-define(INTERVAL, 60).
```

Comment:
Let's add _SEC suffix to make it clear these are seconds

Reply:
or `INTERVAL_IN_SEC`.